src/Frontend/Pages/LandingPage/landing-page-button-container.tsx

```diff
@@ -22,7 +22,7 @@ const LandingPageButtonContainer: React.FC = () => {
         borderTopRightRadius: 100,
       }}
     >
-      <PrimaryButton onPress={() => navigation.navigate("SignUp")}>
+      <PrimaryButton onPress={() => navigation.navigate("DashBoard")}>
```

Comment:
Should navigate to SignUp page and then after SignUp it should take to dashboard